.github/workflows/ci.yml

```diff
@@ -28,11 +28,25 @@ jobs:
     container:
       image: bilelmoussaoui/flatpak-github-actions:gnome-41
       options: --privileged
+    strategy:
+      matrix:
+        arch: [x86_64, aarch64]
+      # Don't fail the whole workflow if one architecture fails
+      fail-fast: false
     steps:
-    - uses: actions/checkout@v2
-    - uses: bilelmoussaoui/flatpak-github-actions/flatpak-builder@v4
-      with:
-        bundle: telegrand.flatpak
-        manifest-path: build-aux/com.github.melix99.telegrand.Devel.json
-        run-tests: true
-        cache-key: flatpak-builder-${{ github.sha }}
+      - uses: actions/checkout@v2
+      # Docker is required by the docker/setup-qemu-action which enables emulation
+      - name: Install deps
```

Comment:
I suggest adding `if: ${{ matrix.arch != 'x86_64' }}` so that this step is ignored for x86_64.